### ISSUE TITLE
Change Normal Distribution Printout when Using Standard Normal Curve

### ISSUE THREAD

**Issue by jargo52** [numworks/epsilon#2155]
Can we edit the normal curve calculation page to display P(z>###) when using a mean of 0 and standard deviation of 1?  It would seemingly be a quick adjustment and allow for slightly more appropriate notation when using the standard normal curve as part of their calculations.

Basically IF(mean=0 and SD=1)THEN(P(z>_____)ELSE(P(x>_____).

Heres a screen shot of the app I am referring to:

![image](https://github.com/numworks/epsilon/assets/160655231/13953655-808f-4ff5-884d-9b7cba971075)
 Since mean and standard deviation are 0 and 1, could we have it display z instead of x?  But continue to use x otherwise?

**Comment by Evangelospro** [comment]
Is there an update on this? I think this is a simple and very intuitive UI change indeed? Is this being worked for in V24 coming this summer?

**Comment by adri1numworks** [comment]
If I understand well, request is to use Z instead of X , for normal distributions of parameters (0,1) ?

**Comment by Evangelospro** [comment]
Yes I think that is done by convention, Z being used for the "standard" normal distribution.
